buffer-api/src/main/java/io/netty/buffer/api/unsafe/UnsafeMemoryManagers.java

```diff
@@ -39,6 +39,11 @@ public MemoryManager getNativeMemoryManager() {
         return new UnsafeMemoryManager(true);
     }
 
+    @Override
+    public String implementationName() {
+        return "Unsafe";
+    }
+
     @Override
     public String toString() {
         return "US";
```

Comment:
US ?

Reply:
"UnSafe". These show up in parameterised test names, so it's useful to have them be short.